modules/harvest/src/HarvestPlanListBuilder.php

```diff
@@ -74,9 +74,9 @@ public function buildRow(EntityInterface $entity) {
     $harvest_plan_id = $entity->get('id')->getString();
     $run_entity = NULL;
 
-    if ($run_id = $this->harvestService->getLastHarvestRunId($harvest_plan_id)) {
+    if ($run_id = $this->harvestService->runRepository->getLastHarvestRunId($harvest_plan_id)) {
```

Comment:
We have a repository in scope here already, right?

Reply:
Good catch... I got too far down the rabbit hole and forgot to look out.  
This is resolved.   I also ended up removing the harvestService because it was no longer used in the listbuilder.